pnet/Private-Networks-PSK-V1.md

```diff
@@ -0,0 +1,75 @@
+## Pre-shared Key Based Private Networks in IPFS
+
+This document describes the first version of private networks (PN) featured in IPFS.
+
+For the first implementation, only pre-shared key (PSK) functionality is available, as the Public Key Infrastructure approach is much more complex and requires more technical preparation.
+
+It was implemented as an additional encryption layer before any IPFS traffic, and was designed to leak the absolute minimum of information on its own. All traffic leaving the node inside a PN is encrypted and there is no characteristic handshake.
```

Comment:
Let's be explicit here on what kind of information is actually leaked, 'minimum' is a relative measure that tells something different to different users.

I would even make it a table with "PSK ON, PSK OFF" and rows describing what can be eavesdropped.

Also, leak has a very negative connotation, let's use words around 'public' or 'publish' instead.